tests/integration/utils.py

```diff
@@ -166,3 +167,15 @@ def validate_json(json_schema, json_data):
     except jsonschema.exceptions.ValidationError:
         return False
     return True
+
+
+def make_list_of_packages_hashable(data):
```

Comment:
When renaming this function, please take into consideration previous comment from @mprahl : https://github.com/release-engineering/cachito/pull/128#pullrequestreview-393941950

Reply:
I tried to name it with an idea it is only for a list of packages. Only to make them hashable. I did renaming because the name was too long for any assertion and the goal was not clear to me from the name before, sorry.